webapp/src/types/api.d.ts

```diff
@@ -62,6 +62,7 @@ export interface Meeting extends Entity {
 }
 
 export interface MeetingNote extends Entity {
+  created_at: string;
```

Comment:
Should these be sorted alphabetically or does that not apply in this context?

Reply:
Yeah, that looks like it was missed in an earlier code change. We don't have a lint rule that enforces it, so it's not worth worrying about. If we add a lint rule about it, it'll get fixed when that rule gets enforced.

Reply:
Okay, that makes sense. Thanks for the explanation.